### PR TITLE
Admin destinations itinerary docs

### PR DESCRIPTION
**Pull request by anna-trofimova**

### CODE REVIEW

File: TripScanner/src/main/java/com/tripscanner/TripScanner/controller/restController/DestinationManagementRestController.java [hunk 1]
```diff
@@ -108,8 +218,31 @@ public ResponseEntity<Destination> uploadImage(@PathVariable long id, @RequestPa
...
     }
 
+
+    @Operation(summary = "Edit destination`s image")
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "200",
+                    description = "Image is edited",
+                    content = {@Content(
+                            mediaType = "application/json",
```
Comment (AdayaUwU) [inline TripScanner/src/main/java/com/tripscanner/TripScanner/controller/restController/DestinationManagementRestController.java:228]: Change mediaType and method's return statement as specified in #134

**Review by AdayaUwU — CHANGES_REQUESTED** [review]
Looks good except for the comments I made:)

File: TripScanner/src/main/java/com/tripscanner/TripScanner/controller/restController/DestinationManagementRestController.java [hunk 2]
```diff
@@ -33,27 +44,89 @@ public class DestinationManagementRestController {
...
         destinationService.save(destination);
         URI location = fromCurrentRequest().path("/{id}").buildAndExpand(destination.getId()).toUri();
         return ResponseEntity.created(location).body(destination);
     }
 
+
+    @Operation(summary = "Edit destination")
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "200",
```
Comment (LovetheFrogs) [inline TripScanner/src/main/java/com/tripscanner/TripScanner/controller/restController/DestinationManagementRestController.java:109]: Change response code to 204

File: TripScanner/src/main/java/com/tripscanner/TripScanner/controller/restController/ItineraryManagementRestController.java [hunk 3]
```diff
@@ -47,8 +86,25 @@ public ResponseEntity<Page<Itinerary>> getItinerary(Pageable pageable) {
     }
 
 
+    @Operation(summary = "Create a new itinerary")
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "200",
```
Comment (LovetheFrogs) [inline TripScanner/src/main/java/com/tripscanner/TripScanner/controller/restController/ItineraryManagementRestController.java:92]: change response code to 201

**Review by LovetheFrogs — APPROVED** [review]

**Pull request merged**